mixins/brand.js

```diff
@@ -20,6 +20,10 @@ export default {
       return setting?.value;
     },
 
+    themeVars() {
+      return getCssThemeVarsFromDocument(`theme-${ this.$store.getters['prefs/theme'] }`);
```

Comment:
This seems like a lot of effort to get the css vars - we already do this in `brand.vue` to get a single value, e.g.

`let uiColor = getComputedStyle(document.body).getPropertyValue('--primary');`

Could we not do the same here? You could then remove the code in css-rule-parser.

We probably only need a single color var for the banner text color